hathor/transaction/resources/push_tx.py

```diff
@@ -101,9 +101,9 @@ def handle_push_tx(self, params: dict[str, Any], client_addr: str) -> dict[str,
             self.manager.push_tx(tx, allow_non_standard_script=self.allow_non_standard_script,
                                  max_output_script_size=self.max_output_script_size)
         except (InvalidNewTransaction, TxValidationError) as e:
-            self.log.warn('push tx rejected', exc_info=True)
             success = False
             message = str(e)
+            self.log.warn('push tx rejected', reason=message)
```

Comment:
I agree that removing the stack trace is good but I think we should improve the error message so we can debug when a transaction fails to be pushed. Maybe add the exception type to the message?

Reply:
I changed it in https://github.com/HathorNetwork/hathor-core/pull/806/commits/bd854c17ae6c8a38ebe99039d4cc1f6a5a27cd69 to use `repr(e)` instead of `str(e)`, which includes the exception type.

Reply:
Thanks!